ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -5068,6 +5068,11 @@ router bgp 65101
    neighbor MPLS-IBGP-PEERS maximum-routes 0
    neighbor MULTIPLE-COMMUNITY peer group
    neighbor MULTIPLE-COMMUNITY send-community standard large
+   neighbor NHP peer group
+   neighbor NHP next-hop-peer
+   neighbor NHP bfd
+   neighbor NHP bfd interval 500 min-rx 500 multiplier 3
+   neighbor NHP password 7 pleaseupdate
```

Comment:
let's remove the password as it will fail in our CI as it is wrong and will raise an issue which does not bring any benefit